fdbserver/include/fdbserver/IKeyValueStore.h

```diff
@@ -133,6 +133,11 @@ class IKeyValueStore : public IClosable {
 	// of a rollback.
 	virtual Future<Void> init() { return Void(); }
 
+	// Obtain the encryption mode of the storage. The encryption mode needs to match the encryption mode of the cluster.
+	virtual Future<EncryptionAtRestMode> encryptionMode() {
```

Comment:
should we rather keep it pure virtual method and force implementations to do the right thing?